drivers/timer/Kconfig

```diff
@@ -51,6 +54,29 @@ endif # APIC_TIMER_TSC
 
 endif # APIC_TIMER
 
+config APIC_TIMER_IRQ_PRIORITY
```

Comment:
I'm wondering if this might be better located inside the `endif # APIC_TIMER` above?

Reply:
It's shared between both drivers, that's why it was moved.  Note that the IRQ one is not, that's the one we tripped over earlier because it doesn't represent an interrupt number at all, it's a kconfig value that MUST conform to the runtime configuration of the CPU (it has to be the IRQ index after the last known io-apic interrupt because that's where our APIC code puts the LVT interrupts)